src/lightning/app/cli/lightning_cli.py

```diff
@@ -121,8 +122,21 @@ def show() -> None:
     pass
 
 
-_main.command()(connect)
-_main.command()(disconnect)
+@_main.group()
+def connect() -> None:
+    """Show resource connections."""
```

Comment:
`Show resource connections` doesn't sound right. Maybe like `Connect apps and data.`

Reply:
Done !